.github/workflows/pull-release.yml

```diff
@@ -0,0 +1,137 @@
+name: get latest helm charts
```

Comment:
What happens when this action fails for some reason? 
Does this job run again or next week?

Reply:
yes, it runs each week irrespective of previous failure.

Reply:
added `workflow_dispatch` trigger nevertheless so we can manually trigger it whenever we want.